pfcli/deployment.py

```diff
@@ -275,15 +275,21 @@ def list(
         secho_error_and_exit("Failed to identify project... Please set project again.")
 
     client: DeploymentClientService = build_client(ServiceType.DEPLOYMENT)
-    deployments = client.list_deployments(str(project_id) if not org else None, False)[
-        "deployments"
-    ]
-    if include_terminated:
+    deployments = client.list_deployments(
+        project_id=str(project_id) if not org else None,
+        archived=False,
+        limit=limit,
+        from_oldest=from_oldest,
+    )
+    num_active_deployments = len(deployments)
+    if include_terminated and limit > num_active_deployments:
         deployments += client.list_deployments(
-            str(project_id) if not org else None, True
-        )["deployments"]
+            project_id=str(project_id) if not org else None,
+            archived=True,
+            limit=limit - num_active_deployments,
+            from_oldest=from_oldest,
+        )
```

Comment:
I am OK with the current order, but eventually `archived=None` field should be available and deployments including both active and archived should be ordered by timestamp.
Currently, it seems to fill the remaining spaces with archived deployments.

Also, it looks good to let users know that the output order prioritizes active deployments. Maybe, we can refine `include_terminated` option description.

Reply:
I agree with your suggestion. I thought that there is no `archived=None` option 😅. I will change it as you said.

Reply:
Oh, I see. Then I will merge this PR as is.